### DISCUSSION TITLE
Tasks in help output + descriptions

### DISCUSSION THREAD

**Discussion by paularmstrong** [moonrepo/moon#1241] [Ideas]
Two ideas, unless I've completely missed some documentation:

1. List tasks in help output.

   ```sh
   moon task --help
   moon <project>: --help
   ```
2. Tasks should have some descriptions and help information. This would help describe to developers what the expected outcome of a task will be.

**Comment by milesj** [comment]
I'll look into this, but because of Rust's static nature, I'm not sure how possible it would be.

At minimum I can probably update `moon task` and `moon project` to include more info.
